templates/OME-TIFF_Conversion.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>I think "Converts a folder of single channel tiffs into a multichannel OME-TIFF." can be moved to right before the last block of code (load_utils.fov_to_ome) to mirror the placement of the comment in the ome-to-single channel section above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>